[A770] New failing tests in the nightly run

https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10072933761/job/27845901887
```
=========================== short test summary info ============================
FAILED language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
= 4 failed, 9479 passed, 22 skipped, 1258 xfailed, 168 warnings in 842.37s (0:14:02) =
```

--- a/scripts/skiplist/a770/language.txt
+++ b/scripts/skiplist/a770/language.txt
@@ -472,3 +472,8 @@ test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-fl
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-float16-float16-1_1]
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-False-none-tf32-float16-float16-1_0]
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-False-none-tf32-float16-float16-1_1]
+# https://github.com/intel/intel-xpu-backend-for-triton/issues/1689
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_0]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_1]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_0]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_1]

